docs/sources/mimir/release-notes/v2.14.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Grafana Mimir version 2.14 release notes"
+menuTitle: "V2.14 release notes"
+description: "Release notes for Grafana Mimir version 2.14"
+weight: 1100
+---
+
+# Grafana Mimir version 2.14 release notes
+
+<!-- vale Grafana.GoogleWill = NO -->
+<!-- vale Grafana.Timeless = NO -->
+<!-- Release notes are often future focused -->
+
+Grafana Labs is excited to announce version 2.14 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release.
+For the complete list of changes, refer to the [CHANGELOG](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+The minimal supported version of Go is updated to 1.22.
+
+The streaming of the chunks from store-gateways to queriers is enabled by default.
+
+Alertmanager adds the `-alertmanager.log-parsing-label-matchers` configuration option to control logging when parsing label matchers.
+This flag is intended to be used with `-alertmanager.utf8-strict-mode-enabled` to validate UTF-8 strict mode is working as intended.
+The default value for the configuration option is `false`.
+
+Alertmanager adds the `-alertmanager.utf8-migration-logging-enabled` configuration option, that enables logging of tenant configurations,
+incompatible with UTF-8 strict mode. The default value for the configuration option is `false`.
+
+Compactor adds a new `cortex_compactor_disk_out_of_space_errors_total` counter metric, that tracks how many times a compaction failed
+due to the compactor being out of disk. A new related alert `MimirCompactorHasRunOutOfDiskSpace` and it's runbook are added.
+
+Incoming OTLP requests were previously size-limited with the distributor's `-distributor.max-recv-msg-size` configuration.
+Distributor has a new `-distributor.max-otlp-request-size` configuration for limiting OTLP requests. The default value is set to limit request to 100 MiB.
+
+Distributor now replies with `Retry-After` header on retryable errors. The feature is controlled by the `-distributor.retry-after-header.enabled` configuration,
+that is enabled by default.
+
```

Comment:
One new feature that we could mention is support for marking ingesters as read-only, that is used in updated downscaling procedure (PR to be merged: https://github.com/grafana/mimir/pull/9208)